src/sonic-yang-models/yang-models/sonic-dash.yang

```diff
@@ -385,7 +385,7 @@ module sonic-dash {
                 }
 
                 leaf vnet {
-                    when "((current()/../action_type = 'vnet') or (current()/../action_type = 'vnet-direct'))";
+                    when "((current()/../action_type = 'vnet') or (current()/../action_type = 'vnet_direct'))";
```

Comment:
Would you please add unit test for this action_type?

Reply:
sure, will add. i would like to unblock the issue first as this is a typo fix.